pkg/utils/client/apply.go

```diff
@@ -305,7 +305,8 @@ func (a *Applier) configFor(gv schema.GroupVersion) (*rest.Config, error) {
 		config.APIPath = "apis/"
 	}
 
-	config.NegotiatedSerializer = serializer.NegotiatedSerializerWrapper(runtime.SerializerInfo{Serializer: unstructured.UnstructuredJSONScheme})
+	contentConfig := resource.UnstructuredPlusDefaultContentConfig()
+	config.NegotiatedSerializer = contentConfig.NegotiatedSerializer
```

Comment:
```suggestion
	config.NegotiatedSerializer = resource.UnstructuredPlusDefaultContentConfig().NegotiatedSerializer
```
Maybe.

Deneb away!

Reply:
It makes the line really long so I'm tempted to skip this suggestion /shrug